spec/translator_spec.lua

```diff
@@ -483,6 +483,7 @@ local m: module = {}
 local xs: {any} = {10, "hello", 3.14}
 
 local function f(x: any, y: any): any
+    return nil as nil
```

Comment:
Could this be just `return nil`, without the `as`?

Reply:
It could be, but I added `as nil` to increase the testing of this test case. As the removal of `as …` syntax was not tested before.

Of course, I believe I could remove this line and the function return type to be closer to the original test case, which would also remove the trailing white space but arguably isn't testing much (such as the type at EOL corner case).

Reply:
A test for EOL stuff sounds like a good idea but I feel it deserves a separate testcase (`it` block with appropriate name)

Reply:
Added testcase in c43d3d1d18f5867d2a2d8566c77aa24e7e8a0210.